chromadb/server/fastapi/__init__.py

```diff
@@ -290,6 +296,24 @@ def heartbeat(self) -> Dict[str, int]:
     def version(self) -> str:
         return self._api.get_version()
 
+    @trace_method("FastAPI.env", OpenTelemetryGranularity.OPERATION)
+    @authz_context(
+        action=AuthzResourceActions.ENV,
+        resource=DynamicAuthzResource(
+            type=AuthzResourceTypes.API,
+        ),
+    )
+    def env(self) -> SystemInfo:
+        _local_env = self._api.env()
+        if not _local_env or not _local_env.get("client"):
+            raise HTTPException(
+                status_code=500,
+                detail="Unable to get system info.",
+            )
+        # this can be done better
+        _local_env["client"]["mode"] = ChromaMode.SERVER_SINGLE_NODE
```

Comment:
I don't understand -- should we be setting this only once somewhere?

Reply:
Actually, when returning the response from the server, we'll get that it is a local persistent client, which returned from the REST API does not look intuitive, hence the remapping within the REST API.

Reply:
I see. Is there some other settings value we can read to determine this? I don't like set-then-read on every request to this endpoint, it feels hacky.